src/components/Docs/components/NodeSnapshots.tsx

```diff
@@ -0,0 +1,115 @@
+// components/NodeSnapshots.tsx
+import axios from "axios";
+import { format } from "date-fns";
+import React, { useEffect, useState } from "react";
+
+interface NodeSnapshotsProps {
+  apiUrl: string;
+}
+
+interface Snapshot {
+  environment: string;
+  type: string;
+  networkVersion?: string;
+  height?: number;
+  creationDate?: string;
+  link?: string;
+}
+
+const NodeSnapshots: React.FC<NodeSnapshotsProps> = ({ apiUrl }) => {
```

Comment:
You should use the `NetworkTypeTabs` component like in `GovUpgradeProposals.tsx`